elephant/signal.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+"""
+docstring goes here.
+:copyright: Copyright 2015 by the Elephant team, see AUTHORS.txt.
+:license: Modified BSD, see LICENSE.txt for details.
+"""
+
+import numpy as np
+import scipy.signal
+import quantities as pq
+import neo
+
+
+def welchpsd(data, num_seg=8, len_seg=None, freq_res=None, overlap=0.5,
+             **kargs):
```

Comment:
I think it would be better to have them as explicit keyword arguments, with documentation.